xbmc/pvr/windows/GUIWindowPVRBase.h

```diff
@@ -84,7 +84,7 @@ namespace PVR
 
     bool InitChannelGroup(void);
     virtual CPVRChannelGroupPtr GetChannelGroup(void);
-    virtual void SetChannelGroup(const CPVRChannelGroupPtr &group);
+    void SetChannelGroup(const CPVRChannelGroupPtr &group, bool bUpdate = true);
```

Comment:
Sure, can do. Sigh, I wish my predecessors would have been more professional on this. But ofc this is not excuse for not adding docs for new stuff.